packages/ui-react/src/pages/User/__snapshots__/User.test.tsx.snap

```diff
@@ -101,13 +101,16 @@ exports[`User Component tests > Account Page 1`] = `
     <div
       class="_mainColumn_e9c5ca"
     >
-      <div
+      <section
+        aria-labelledby="account.about_you_1235"
```

Comment:
Dots are also not allowed in an id attribute. This is also something we need to fix. I do not know the cause of this.

Reply:
I think the cause is a that translations values are not loaded and the keys are used instead.
It's ok to still use the keys. But we should remove the dots as they could cause unexpected behaviour. Probably not for snapshot comparison, but in other situations that may arise.

Reply:
- [x] Resolved in another [comment](https://github.com/Videodock/ott-web-app/pull/15#discussion_r1447496364)